EDITORS.md

```diff
@@ -61,102 +61,10 @@ mason_lspconfig.setup_handlers {
 }
 ```
 
-### Neovim Limitations
+### Limitations
 
 Ruby LSP only supports pull diagnostics, and neovim versions prior to v0.10.0-dev-695+g58f948614 only support [publishDiagnostics](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_publishDiagnostics).
```

Comment:
Discussed with @vinistock and decided that recommending a plugin for something temporary might be overkill.

This line and the comment in the snippet below already makes it clear that v0.10 is still in dev, so I'll close this PR, and update the snippet in the [other PR](https://github.com/Shopify/ruby-lsp/pull/1936).